packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -44,7 +44,10 @@ const LightboxMediaViewer = ({ media, ...modalProps }) => {
                 {media.type === 'video' ? (
                   <VideoPlayer videoId={media.src} />
                 ) : (
-                  <Image defaultSrc={media.src} alt={media.alt} />
+                  <div
+                    className={`${prefix}--lightbox-media-viewer__media-image-wrapper`}>
```

Comment:
@RaphaelAmadeu I was looking around and couldn't find this class, is this being used?

Reply:
@jeffchew ,
This was part of a fix try,
I've removed in on the latest commit,
Thank you!